### PR TITLE
fix(source-google-ads): updated handling of 500 error

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What
oncall https://github.com/airbytehq/oncall/issues/6142

## How
updated error message for 500 status code
image

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py [hunk 1]
```diff
@@ -20,6 +20,16 @@
 API_VERSION = "v15"
 
 
+def on_give_up(details):
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py:23]: Should we have a test for this change?

**Review by tolik0 — APPROVED** [review]

**Review by maxi297 — COMMENTED** [review]
Just want to make sure we create a meaningful safety net before approving

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py (hunk 1 above)
Comment (darynaishchenko) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py:23]: working on it

Comment (darynaishchenko) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py:23]: @maxi297, Updated unit tests

**Review by maxi297 — APPROVED** [review]
Awesome, thanks!

**Review by bazarnov — APPROVED** [review]

**Pull request merged**